dev/tools/controllerbuilder/generate.sh

```diff
@@ -227,7 +227,14 @@ go run . generate-mapper \
    --output-dir $REPO_ROOT/pkg/controller/direct/ \
    --api-dir $REPO_ROOT/apis/
 
-
+go run . generate-direct-reconciler \
```

Comment:
This is a new SciFi resource that I'm currently working on. This is to give examples on how to use the new command since we don't have  document yet.

Reply:
yes. though the stout messages are not aligned among the nested commands. We could fix that